doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,1052 @@
+
+* :ref:`lib_location` library:
+
+  * Removed PoLTE support as the service is discontinued.
+
+* :ref:`lib_multicell_location` library:
+
+  * Added:
+
+    * Timeout parameter.
+    * Structure for input parameters for :c:func:`multicell_location_get` to make updates easier in the future.
+
+  * Updated the conversions of RSRP and RSRQ.
+    These now use common macros that follow the conversion algorithms defined in the `AT Commands Reference Guide`_.
+  * Removed PoLTE support as the service is discontinued.
+
+* :ref:`lib_rest_client` library:
+
+  * Updated:
+
+    * Timeout handling.
+      Now using http_client library timeout also.
+    * A zero timeout value is now handled as "no timeout" (wait forever) to avoid immediate timeouts.
+
+  * Removed ``CONFIG_REST_CLIENT_SCKT_SEND_TIMEOUT`` and ``CONFIG_REST_CLIENT_SCKT_RECV_TIMEOUT`` Kconfig options.
+
+* :ref:`lib_nrf_cloud_rest` library:
+
+  * Updated the :c:func:`nrf_cloud_rest_send_location` function to accept a :c:struct:`nrf_cloud_gnss_data` pointer instead of an NMEA sentence.
+
+* :ref:`lib_nrf_cloud_pgps` library:
+
+  * Added:
+
+    * :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_DOWNLOAD_TRANSPORT_HTTP` and :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_DOWNLOAD_TRANSPORT_CUSTOM` Kconfig options.
+    * :c:func:`nrf_cloud_pgps_begin_update` function that prepares the P-GPS subsystem to receive downloads from a custom transport.
+    * :c:func:`nrf_cloud_pgps_process_update` function that stores a portion of a P-GPS download to flash.
+    * :c:func:`nrf_cloud_pgps_finish_update` function that a user of the P-GPS library calls when the custom download completes.
+
+  * Reduced logging level for many messages to debug (``DBG``).
+
+* :ref:`lib_azure_iot_hub` library:
+
+  * Reworked to use `Azure SDK for Embedded C`_.
+  * Updated the APIs for both IoT Hub and DPS interaction.
+    The applications and samples that use the library have been updated accordingly.
+
+* :ref:`lib_download_client` library:
+
+  * Fixed:
+
+    * Handling of duplicated CoAP packets.
+    * Handling of timeout errors when using CoAP.
+
+Other libraries
+---------------
+
+* Added:
+
+  * Documentation for the :ref:`lib_adp536x` library.
+  * Documentation for the :ref:`lib_flash_map_pm` library.
+  * :ref:`lib_identity_key` library.
+
+* :ref:`lib_flash_patch` library:
+
+  * Added documentation page.
+  * Modified the :kconfig:option:`CONFIG_DISABLE_FLASH_PATCH` Kconfig option, so that it can be used on the nRF52833 SoC.
+
+* :ref:`doc_fw_info` module:
+
+  * Fixed a bug where MCUboot would experience a fault when using the :ref:`doc_fw_info_ext_api` feature.
+
+* :ref:`nrf_rpc_ipc_readme`:
+
+  * This is a new library.
+    It can use different transport implementation for each nRF RPC group.
+  * Memory for remote procedure calls is now allocated on a heap instead of the calling thread stack.
```

Comment:
@KAGA164 , @peknis , this library went in in https://github.com/nrfconnect/sdk-nrf/pull/7510 . Two questions:

1. Are there going to be more libraries for nRF RPC? If not, the library should be moved to Other libraries and 1-entry section should be removed.
2. Is this a brand new library? If so, there is no point in listing fixes to this library in the release notes where it was introduced.

Reply:
This is a single library only we only plan to implement different transport layers for it.
It is not brand new library, we released it in the NCS v1.8 probably